.github/workflows/lint.yml

```diff
@@ -4,16 +4,27 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - 'main'
+      - 'master'
```

Comment:
Since we are updating this, what's the rationale for running the linter check on push? All pushes are fornted by pull requests which are blocked on linter fails. So why not just drop the linter on push?

Reply:
updated PTAL

/unhold